### ISSUE TITLE
Cellular Pin Code Rejected 

### ISSUE THREAD

**Issue by swvalenti194** [iamromulan/quectel-rgmii-configuration-notes#12]
I keep getting pin code rejected on the r520gl.  It sees the provider but for some reason it throws that error.  Any idea on how to resolve?  Thanks

**Comment by iamromulan** [comment]
Pin code? Where does it show this?

**Comment by chmanie** [comment]
Yes, I'm seeing this too. It's in OpenWRT when trying to connect the RM520N-GL using MBIM:

![Image](https://github.com/user-attachments/assets/0ff56b68-8bbc-4f09-ae05-d3020439ba80)

**Comment by iamromulan** [comment]
No idea. I have very little experience working with it over MBIM from a host device. Closing for now.
